src/main/webapp/resources/js/pages/admin/components/statistics/AdvancedStatistics.jsx

```diff
@@ -3,7 +3,7 @@
  * which displays statistics in charts
  */
 
-import { Bar, Column, Line, Pie } from "@ant-design/charts";
+import { Bar, Column, Line, Pie } from "@ant-design/plots";
```

Comment:
`Plots` is lighter weight and contains all the components we were using.

Reply:
Nice seems to work just fine.  Thanks